src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -345,6 +345,7 @@ typedef struct {
     struct fid_cq *p2p_cq;
     struct fid_cntr *rma_cmpl_cntr;
     struct fid_stx *stx_ctx;    /* shared TX context for RMA */
+    struct fid_ep *rma_sep;     /* dedicated scalable EP for RMA */
```

Comment:
`stx_ctx` is used only in RMA, maybe we want to change its name to `rma_ stx_ctx`.